apps/nekohit/src/app/create-project/create-project.component.html

```diff
@@ -96,28 +96,23 @@
               </label>
               <div class="">
                 <p-inputNumber
-                  formControlName="securityStake"
+                  formControlName="securityStakePercent"
                   mode="decimal"
-                  [minFractionDigits]="
-                    tokenService.getTokenByHash(token.value).decimals
-                  "
-                  [maxFractionDigits]="
-                    tokenService.getTokenByHash(token.value).decimals
-                  "
-                  [suffix]="
-                    ' ' + tokenService.getTokenByHash(token.value).symbol
-                  "
-                  [max]="
-                    getTokenBalance(
-                      tokenService.getTokenByHash(token.value).symbol
-                    )
-                  "
-                  [min]="0"
+                  [minFractionDigits]="2"
+                  [maxFractionDigits]="2"
+                  suffix="%"
+                  [min]="1"
                   [showButtons]="true"
                 ></p-inputNumber>
 
                 <div class="m-2 font-italic text-sm">
-                  ≈ ${{ s.selectedTokenPrice * securityStake | number }}
+                  =
+                  {{ (fundingGoal * securityStakePercent) / 100 }}
```

Comment:
Another extreme situation (sorry for too many extreme cases) is shown in the pic. But this doesn't affect the contract invocation, so if it's tricky to fix, just leave it there. For this situation, anyone who knows how floating number works will understand🙂

![image](https://user-images.githubusercontent.com/8587572/148631024-8578c14a-78a5-47b7-9d4a-6ce594776cc3.png)

Reply:
the only thing that comes to my mind is displaying a warning that the decimals after decimals after the 8 will be ignored.

Reply:
or maybe even better would limit the decimals to 2 in case of CAT token

Reply:
ill take a look, so I will put this PR to WIP for now

Reply:
I think there is a way to obtain how many digits a given token has. And it can be an optional parameter of the `number` pipe to format (so it won't break the existing usage), or, floor the input number, since the contract will calculate the staked amount in integer, which is the `floor` of the result.

----

Hmmmm, `floor` might not be an accurate word to use, since it is used when transforming a floating number into an integer. To format N digits floating number into M digits (N > M), it would be cut off, by discarding anything beyond the M-th digits, like transforming `0.123456789` into `0.12345`:
```javascript
let num = 0.123456789;
num = num.toFixed(5);  // num would be 0.12345
```

Reply:
I use now the existing customPipe tokenNumber: "symbol" .. this seems to display it correctly.